src/test/ts/browser/EditorInitTest.ts

```diff
@@ -1,15 +1,15 @@
 import { Assertions, Chain, GeneralSteps, Logger, Pipeline } from '@ephox/agar';
 import { UnitTest } from '@ephox/bedrock-client';
-import { Element } from '@ephox/dom-globals';
 import { ApiChains } from '@ephox/mcagar';
 import { VersionLoader } from '@tinymce/miniature';
+import { Editor as TinyMCEEditor } from 'tinymce';
 
 import { cRemove, cRender, cDOMNode, cEditor, cReRender } from '../alien/Loader';
 
 UnitTest.asynctest('Editor.test', (success, failure) => {
-  const cAssertProperty = (propName: any, expected: any) => {
-    return Chain.op((el: any) => {
-      Assertions.assertEq(propName + ' should be ' + expected, expected, el[propName]);
+  const cAssertProperty = (propName: string, expected: string) => {
```

Comment:
Would `propName: keyof HTMLElement` avoid the cast?

Reply:
Sadly no (I tried that too). The problem is that we try to get the `name` property which is only on form elements not HTMLElement element. Unfortunately we can't specalise it to only working with form elements because we need to test on divs as well (in which case we don't get "name").